xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -533,12 +533,18 @@ void CActiveAE::StateMachine(int signal, Protocol *port, Message *msg)
         switch (signal)
         {
         case CActiveAEControlProtocol::RECONFIGURE:
+        {
           if (m_streams.empty())
           {
             streaming = false;
             m_sink.m_controlPort.SendOutMessage(CSinkControlProtocol::STREAMING, &streaming, sizeof(bool));
           }
+          AudioSettings settings = m_settings;
           LoadSettings();
+          if (settings.streamNoise != m_settings.streamNoise)
```

Comment:
My idea: Only if the setting has changed, we need to send the signal. Intention: Avoid GenerateNoise() if not needed.